R/process_data.R

```diff
@@ -181,6 +186,73 @@ set_initial_plan_carsten_missings_to_zero <- function(data,
   return(data)
 }
 
+#' Remove rows from PACTA results that belong to company-sector combinations
+#' for which there is no positive production value in the relevant year of
+#' exposure (last year of forecast). This handles the edge case that a company
+#' may have a positive exposure for this sector, but none of the technologies
+#' covered in this analysis have any positive production. Such inconsistencies
+#' may arise e.g. because of unclear separation of the LDV and HDV sectors.
+#'
+#' @inheritParams calculate_annual_profits
+#' @inheritParams report_company_drops
+#' @param data tibble containing filtered PACTA results
+#'
+#' @return A tibble of data without rows with no exposure info
+#' @noRd
+remove_sectors_with_missing_production <- function(data,
```

Comment:
I would suggest to add unit tests, 1 where no companies are removed and 1 where companies are removed